src/controllers/StoryController.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+namespace App\Controllers;
+
+use App\Exceptions\StoryNotFoundException;
+use App\Models\Story;
+use App\Models\Comment;
+
+class StoryController
+{
+
+    protected $storyModel;
+    protected $commentModel;
+
+    public function __construct($config)
+    {
+        $this->storyModel   = new Story($config);
+        $this->commentModel = new Comment($config);
+    }
+
+    public function index()
+    {
+        if (!isset($_GET['id'])) {
+            header("Location: /");
+            exit;
+        }
+
+        try {
+            $story = $this->storyModel->show((int)$_GET['id']);
+        } catch (StoryNotFoundException $e) {
+            header("Location: /");
+            exit;
+        }
+
+        $comments = $this->commentModel->byStory($story['id']);
```

Comment:
Does Comment::byStory() let you search on other components of the story (headline, url, etc)? If not, a more descriptive name might include the "Id" field.